src/components/Typography/index.tsx

```diff
@@ -286,24 +286,17 @@ const TextWithTooltip = ({
 
   return (
     <Tooltip text={finalStringChildren}>
-      {({
-        onMouseEnter = () => {},
-        onFocus = () => {},
-        ...tooltipProps
-      }: {
-        onMouseEnter: MouseEventHandler<Element>
-        onFocus: FocusEventHandler<Element>
-      }) => (
+      {({ onMouseEnter, onFocus, ...tooltipProps }) => (
         <Text
           {...props}
           onMouseEnter={ev => {
-            if (isTruncated(ev.currentTarget)) {
-              onMouseEnter(ev)
+            if (isTruncated(ev.currentTarget) && onMouseEnter) {
+              onMouseEnter?.(ev)
             }
           }}
           onFocus={ev => {
-            if (isTruncated(ev.currentTarget)) {
-              onFocus(ev)
+            if (isTruncated(ev.currentTarget) && onFocus) {
+              onFocus?.(ev)
```

Comment:
What's the reason to check if these callbacks are defined, and then optional-chain the function call?